accessibility.sh

```diff
@@ -0,0 +1 @@
+npx -y replace-in-files-cli --regex="outline: (0|none);" --replacement="/* outline: $& */" bower_components/
```

Comment:
A bunch of old components are disabling outlines, which is a big no-no (https://www.outlinenone.com). I first tried updating them but there were a lot of breaking changes that I didn't have time to resolve. This [`package-package` equivalent script](https://www.npmjs.com/package/patch-package) instead just comments out the offending styles inline.